Source/WebKit/UIProcess/API/Cocoa/WKContentRuleList.mm

```diff
@@ -70,4 +71,18 @@ + (BOOL)_supportsRegularExpression:(NSString *)regex
 #endif
 }
 
++ (NSError *)_parseRuleList:(NSString *)ruleList
+{
+#if ENABLE(CONTENT_EXTENSIONS)
+    std::error_code error = API::ContentRuleList::parseRuleList(ruleList);
+    NSInteger errorCode = error.value();
+    if (!errorCode)
+        return nil;
+
+    return [NSError errorWithDomain:WKErrorDomain code:errorCode userInfo:nil];
```

Comment:
This error code is not in WKErrorDomain.  Use WKErrorContentRuleListStoreCompileFailed and maybe put more info in the userInfo like we do in compileContentRuleListForIdentifier:

Reply:
Sounds good - I see that example.